packages/core/src/App/Components/Layout/Footer/__tests__/network-status.spec.tsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import userEvent from '@testing-library/user-event';
 import { render, screen } from '@testing-library/react';
-import { NetworkStatus } from '../network-status.jsx';
+import NetworkStatus from '../network-status.jsx';
+import { StoreProvider, mockStore } from '@deriv/stores';
 
+const store = mockStore();
 const MockNetworkStatus = ({ is_mobile = true }) => (
-    <NetworkStatus status={{ class: 'offline' }} is_mobile={is_mobile} />
+    <StoreProvider store={store}>
+        <NetworkStatus is_mobile={is_mobile} />
```

Comment:
@yaswanth-deriv, just to clarify, isn't `status={{ class: 'offline' }}` still being used?

```suggestion
        <NetworkStatus is_mobile={is_mobile} />
```